addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3458,9 +3458,10 @@ export class OdooEditor extends EventTarget {
             // just its rows.
             rangeContent = tableClone;
         }
-        if (rangeContent.firstChild.nodeName === 'TABLE') {
+        const table = closestElement(range.startContainer, 'table');
+        if (rangeContent.firstChild.nodeName === 'TABLE' && table) {
             // Make sure the full leading table is copied.
-            rangeContent.firstChild.after(closestElement(range.startContainer, 'table').cloneNode(true));
+            rangeContent.firstChild.after(table.cloneNode(true));
```

Comment:
It is not clear to me how `closestElement(range.startContainer, 'table')` could not be found if `rangeContent.firstChild.nodeName === 'TABLE'` is `true`. Could you write a test for this case please ? This way I will be able to see the practical case that can create that situation and also it will be properly tested which is good.

Reply:
@dmo-odoo I've added a new test case for the above scenario. Please check it out. Thank you!

Reply:
@dhba-odoo Thank you, now I understand the special case ! It requires a reverse selection that goes outside of the table.